index.js

```diff
@@ -4,7 +4,40 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+   //console.log(process.argv)
+    
+   if(!process.argv[3]) {
+        
+    if(!process.argv[2]) {
+        return "No operation provided...";
+}
+return "No numbers provided...";
+}
+if( process.argv[2] === 'plus'){
```

Comment:
The code in this if block and the one on line 26 are very similar with few differences. How can you refactor this code to be less repetitive??